src/platform/amd/acp_6_3/lib/dai.c

```diff
@@ -72,6 +72,105 @@ static struct dai hsdai[] = {
 	}
 };
 
+static struct dai swaudiodai[] = {
+	{
+		.index = DI_SDW0_ACP_SW_AUDIO_TX,
+		.plat_data = {
+			.base = DAI_BASE_REM,
+			.fifo[SOF_IPC_STREAM_PLAYBACK] = {
+			.offset         = DAI_BASE_REM + SW0_AUDIO_TX_FIFO_OFFST,
+			.depth          = 8,
```

Comment:
Is thsis compatible with the definition in the previous patch 
````#define SW0_AUDIO_FIFO_SIZE			128````

Reply:
register offset

Reply:
depth not being used, removed.